Src/MakingSense.SafeBrowsing/GoogleSafeBrowsing/GoogleSafeBrowsingUpdater.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Google.Apis.Safebrowsing.v4;
+using Google.Apis.Safebrowsing.v4.Data;
+using Google.Apis.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MakingSense.SafeBrowsing.GoogleSafeBrowsing
+{
+    /// <summary>
+    /// Allow to download Google Safe Browsing lists from HTTP and update them
+    /// </summary>
+    public class GoogleSafeBrowsingUpdater : IUpdater
+    {
+        private readonly GoogleSafeBrowsingConfiguration _configuration;
+        private readonly GoogleSafeBrowsingDatabase _database;
+        private readonly SafebrowsingService _safebrowsingService;
+
+        private const string ANY_PLATFORM = "ANY_PLATFORM";
+        private const string URL = "URL";
+        private const string RAW = "RAW";
+        private const string FULL_UPDATE = "FULL_UPDATE";
+
+        /// <summary>
+        /// Creates new instance and initializes configuration parameters
+        /// </summary>
+        /// <param name="apiKey">Google API KEY</param>
+        /// <param name="clientId">Unique client ID</param>
+        /// <param name="clientVersion">Client version</param>
+        /// <param name="region">Geographic location in ISO 3166-1 alpha-2 format</param>
+        public GoogleSafeBrowsingUpdater(string apiKey, string clientId, string clientVersion, string region = null)
```

Comment:
What do you think about to receive a configuration object in place of primitives?